lib/tasks/demo.rake

```diff
@@ -118,6 +118,7 @@ namespace :demo do
 
     days = (ENV['DAYS'] || 14).to_i
     count = (ENV['COUNT'] || 50).to_i
+    skip_daily_analytics_update = (ENV['SKIP'] || 'false')
```

Comment:
I think if you do something like (ENV['SKIP_ANALYTICS'] != 'true'), then you won't have to do a string comparison for 'false' on line 299

Reply:
good idea, thanks!